index.js

```diff
@@ -195,13 +195,16 @@ class EpicGamesClientLoginAdapter {
     const pendingDocument = new PendingXHR(page);
     pendingDocument.resourceType = 'document';
     await page.goto('https://epicgames.com/id');
-    await page.waitFor(1000);
-    await pendingXHR.waitOnceForAllXhrFinished();
-    await page.waitFor(1000);
-    await pendingDocument.waitOnceForAllXhrFinished();
-
-    if (page.url() != this.ACCOUNT_PAGE) {
-      await this.authenticate(credentials, page, options);
+    const login = credentials.login || credentials.email || credentials.username;
+    if (login && credentials.password) {
+      const loginWithEpicButton = await page.waitForSelector('#login-with-epic');
+      await loginWithEpicButton.click();
+      const usernameOrEmailField = await page.waitForSelector('#email');
+      await usernameOrEmailField.type(login, { delay: options.inputDelay });
+      const passwordField = await page.waitForSelector('#password');
+      await passwordField.type(credentials.password, { delay: options.inputDelay });
+      const loginButton = await page.waitForSelector('#sign-in:not(:disabled)');
+      await loginButton.click();
```

Comment:
This is strange behaviour with merge conflict resolver

Reply:
merge conflict resolver?

Reply:
Yup. This is sync with base repo